at-hoangtran/Week4/Final/Controllers/session_controller.rb

```diff
@@ -0,0 +1,64 @@
+require_relative '../Migrate/connect_db'
+require_relative '../Models/employee_model'
+require 'pry'
+require 'bcrypt'
+class SessionController
+  $session = {}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/8bc115c94428b11d6e3406f6ef06575baf3e6ffc/at-hoangtran/Week4/Final/Controllers/session_controller.rb#L6"></span>Style/GlobalVars: Do not introduce global variables. (https://github.com/rubocop-hq/ruby-style-guide#instance-vars, http://www.zenspider.com/Languages/Ruby/QuickRef.html)</td>
    </tr>
  </tbody>
</table>